### PR TITLE
[ONNX] Replace decomposeLinear pre process pass with a symbolic

### PR DESCRIPTION
**Pull request by KsenijaS**
Replace decomposeLinear pre process pass with a symbolic

### CODE REVIEW

**Review by jiafatom — COMMENTED** [review]
Why we want to replace this preprocess with a symbolic? What is the benefit here?

**Review by BowenBao — COMMENTED** [review]
Can you add unit test in `test_pytorch_onnx_onnxruntime.py`?

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -2918,3 +2918,18 @@ def __range_length(g, lo, hi, step):
     sub = g.op("Sub", hi, lo)
     div = g.op("Ceil", true_divide(g, sub, step))
     return g.op("Cast", div, to_i=sym_help.cast_pytorch_to_onnx['Long'])
+
+
+def linear(g, input, weight, bias):
+    rank = sym_help._get_tensor_rank(input)
+    weight = t(g, weight)
+    if rank == 2 and not bias.node().mustBeNone():
```
Comment (shubhambhokare1) [inline torch/onnx/symbolic_opset9.py:2926]: Does this account for cases where the value of rank might be None?

Comment (KsenijaS) [reply torch/onnx/symbolic_opset9.py:2926]: if rank is None then else branch will be executed

**Review by BowenBao — APPROVED** [review]

**Pull request merged**